node/src/actor_system_factory.rs

```diff
@@ -1936,169 +2087,134 @@ mod tests {
-        let mut prepare_initial_messages_params =
-            prepare_initial_messages_params_arc.lock().unwrap();
-        let (
-            main_cryptde_after,
-            alias_cryptde_after,
-            bootstrapper_config_after,
-            actor_factory_after,
-            persistent_config_after,
-        ) = prepare_initial_messages_params.remove(0);
-        assert!(prepare_initial_messages_params.is_empty());
-        let main_cryptde_public_key_after =
-            public_key_for_dyn_cryptde_being_null(main_cryptde_after.as_ref());
-        assert_eq!(
-            main_cryptde_public_key_after,
-            main_cryptde_public_key_before
-        );
-        let alias_cryptde_public_key_after =
-            public_key_for_dyn_cryptde_being_null(alias_cryptde_after.as_ref());
-        assert_eq!(
-            alias_cryptde_public_key_after,
-            alias_cryptde_public_key_before
+    #[test]
+    fn blockchain_bridge_connectivity_check() {
+        let data_dir = ensure_node_home_directory_exists(
+            "actor_system_factory",
+            "blockchain_bridge_connectivity_check",
         );
-        assert_eq!(
-            bootstrapper_config_after.data_directory,
-            irrelevant_data_dir
+        let conn = DbInitializerReal::default()
+            .initialize(&data_dir, DbInitializationConfig::test_default())
+            .unwrap();
+        let mut persistent_config = PersistentConfigurationReal::from(conn);
+        let gas_price = 444;
+        persistent_config.set_gas_price(gas_price).unwrap();
+        let port = find_free_port();
+        let server_url = format!("http://{}:{}", &Ipv4Addr::LOCALHOST.to_string(), port);
+        let test_server = TestServer::start(
+            port,
+            vec![br#"{"jsonrpc":"2.0","id":0,"result":someGarbage}"#.to_vec()],
         );
-        assert_eq!(
-            bootstrapper_config_after.db_password_opt,
-            Some("chameleon".to_string())
+        let wallet = make_wallet("abc");
+        let mut bootstrapper_config = BootstrapperConfig::new();
+        let blockchain_service_url_opt = Some(server_url);
+        bootstrapper_config
+            .blockchain_bridge_config
+            .blockchain_service_url_opt = blockchain_service_url_opt;
+        bootstrapper_config.blockchain_bridge_config.chain = TEST_DEFAULT_CHAIN;
+        bootstrapper_config.data_directory = data_dir;
+        let (addr_tx, addr_rv) = bounded(1);
+        let system = System::new("blockchain_bridge_connectivity_check");
+        let subject = ActorFactoryReal {};
+
+        subject.make_and_start_blockchain_bridge(
+            &bootstrapper_config,
+            &SubsFactoryTestAddrLeaker {
+                address_leaker: addr_tx,
+            },
         );
+
+        let blockchain_addr = addr_rv.try_recv().unwrap();
+        //the message stops also the system
+        blockchain_addr
+            .try_send(assertion_msg_to_test_blockchain_bridge_connectivity(
```

Comment:
Think about using a public manifest constant instead of a function call. It allows to initiate a struct in its literal form which is how we usually construct the message.